packages/protocol/deployments/arbitrum/BorrowingVault-WETHUSDT-1

```diff
@@ -0,0 +1 @@
+0x404CE931E95Bd8B53433e5f96e0F678f3C8bee9A
```

Comment:
no contract at this address

Reply:
? on Arbitrum?
![image](https://github.com/Fujicracy/fuji-v2/assets/32775237/4e28be95-700d-4fca-8fd0-d97c0f0c55ac)